UrologicalSupplies/R4/files/UrologicalSuppliesPrepopulation-0.1.0.cql

```diff
@@ -9,8 +9,12 @@ codesystem "SNOMED-CT": 'http://snomed.info/sct'
 codesystem "HCPCS": 'https://bluebutton.cms.gov/resources/codesystem/hcpcs'
 
 valueset "Urinary Incontinence or Chronic Urinary Retention Disorder": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.70'
-// TODO: Update HCPCS codes after VSAC value set is done
-valueset "Urinary Collection and Irrigation System Device or Supply": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.49'
+//valueset "Administrative Gender (HL7 V3)": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.1.11.1'
```

Comment:
Why is this commented out? Do we need it?

Reply:
Nope I removed it